### ISSUE TITLE
website components now clickable

### ISSUE THREAD

**Issue by spxneo** [DeFUCC/gun-vue#41]
on the page where you list the components whe ni click them i cannot view it

https://gun-vue.js.org/docs/packages/components.html

**Comment by davay42** [comment]
Yes, that's just a list of them with no pages made for each of them. I haven't enough time and will to make demos and docs for each of the components yet. Hope the code is readable enough to use them right away or just copy and go down to using just the composables in your code. Any help appreciated BTW!
